sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -241,9 +245,9 @@ export class ServiceBusManagementClient extends ServiceClient {
       requestPolicyFactories.push(proxyPolicy(options.proxySettings));
     }
     const serviceClientOptions: ServiceClientOptions = {
-      requestPolicyFactories: requestPolicyFactories
+      requestPolicyFactories: requestPolicyFactories,
+      userAgent: userAgent
```

Comment:
We are specifying userAgent here and also adding the userAgentPolicy to the `requestPolicyFactories`. Do we have to do both?

Reply:
Good question.

Seems like `core-http` adds the `userAgentPolicy` as part of creating the default request policy factories (with the userAgent that is provided in the options).

![image](https://user-images.githubusercontent.com/10452642/87739030-a7f20f00-c793-11ea-85ef-e0b59892d884.png)

Having it in the options seems enough then, no need to add `userAgentPolicy` on our own.

Reply:
`options.userAgent` is only used when you call `createDefaultRequestPolicyFactories()`, which I don't think is the case here.

Reply:
Thanks for the reply, @jeremymeng.
I was just about to tag you here. 😁

So, the right thing here would be to add `userAgentPolicy` to the `requestPolicyFactories`, it doesn't matter if I set the userAgent in the options and hence, would remove it.

Is there a way to test if I'm doing the right thing? Is it just console-logging the request to verify if the user-agent string is set on the request?

Reply:
It should show up when doing INFO level logging by setting the environment variables

Reply:
Okay, keeping them both then.

Reply:
I'm logging the request at core-http before sending and I don't see the user-agent header.